src/pools/thread/fixed.ts

```diff
@@ -81,4 +82,14 @@ export class FixedThreadPool<
     // Listen worker messages.
     this.registerWorkerMessageListener(worker, super.workerListener())
   }
+
+  /** @inheritdoc */
+  public get type (): PoolType {
+    return PoolType.FIXED
+  }
+
+  /** @inheritdoc */
+  public get busy (): boolean {
+    return this.internalGetBusyStatus()
```

Comment:
So the performance overhead will be mainly on the fixed pools right ?  I was thinking if is really useful for end users to know if all the workers are busy ? What they can do in that case I mean ?

Reply:
The job admission can't currently be regulated by users. I, for example, use a homemade queue at job admission and I need to trigger it only if needed.